flipper-plugin-realm/src/components/DataTable.tsx

```diff
@@ -20,39 +24,41 @@ export type ColumnType = {
   isPrimaryKey: boolean;
 };
 
-type PropertyType = {
+type DataTableProps = {
   columns: ColumnType[];
-  objects: RealmObject[];
-  schemas: SchemaObject[];
-  currentSchema: SchemaObject;
-  sortingDirection: 'ascend' | 'descend' | null;
+  objects: IndexableRealmObject[];
+  schemas: SortedObjectSchema[];
+  currentSchema: Realm.CanonicalObjectSchema;
+  sortingDirection?: 'ascend' | 'descend' | null;
   sortingColumn: string | null;
   generateMenuItems?: MenuItemGenerator;
   style?: Record<string, unknown>;
   setdropdownProp: Function;
   dropdownProp: Object;
-  scrollX: number;
-  scrollY: number;
+  scrollX?: number;
+  scrollY?: number;
   enableSort: boolean;
   hasMore: boolean;
   totalObjects?: number;
   fetchMore: () => void;
   setNewInspectionData: (
     inspectionData: InspectionDataType,
-    wipeStacks?: boolean
+    wipeStacks?: boolean,
   ) => void;
-  clickAction?: (object: RealmObject) => void;
+  clickAction?: (object: IndexableRealmObject) => void;
 };
 
 type ClickableTextType = {
-  displayText: string;
+  displayText: string | number | JSX.Element;
```

Comment:
If `displayText` is more than a string, perhaps a better name would be `displayContent`

Reply:
I could call it `displayValue`, kind of provides good relationship with `value` which `ClickableText` has as well.

Reply:
I'm happy with `displayValue`